class/deeplink-schema-match.ts

```
@@ -32,336 +16,21 @@
-    return text.startsWith('widget?action=');
-  }
 
-  static hasNeededJsonKeysForMultiSigSharing(str: string): boolean {
-    let obj;
-
-    // Check if it's a valid JSON
-    try {
-      obj = JSON.parse(str);
-    } catch (e) {
-      return false;
+    if (this.isBothBitcoinAndLightning(url)) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)